packages/corelib/package.json

```diff
@@ -44,7 +44,7 @@
 		"fast-clone": "^1.5.13",
 		"i18next": "^21.9.1",
 		"influx": "^5.9.3",
-		"mos-connection": "^2.0.1",
+		"@mos-connection/model": "3.0.0-nightly-feat-mono-repo-20221216-111645-0e02d70.0",
```

Comment:
do we need this dependency here? it is also included in shared-lib, so perhaps can be accessed via there?

Reply:
Hm, but that would be an "extraneous import" wouldn't it?

Reply:
I mean make shared-lib re-export it (like https://github.com/nrkno/sofie-core/blob/release49/packages/shared-lib/src/tsr.ts).

We do have the ci job which will make sure all versions of `@mos-connection/model` being used are the same, which should stop that for corelib. But if the blueprints have to define `@mos-connection/model` explicitly, then there will be a high risk of it using two different versions